src/components/WorkOrders/__snapshots__/JobsTable.test.js.snap

```diff
@@ -132,6 +132,9 @@ exports[`JobsTable component should render properly 1`] = `
         </tr>
       </tbody>
     </table>
+    <div
+      class="page-navigation govuk-!-padding-bottom-5"
+    />
```

Comment:
You need to add `pageNumber` and `handlePageClick` props to `JobsTable.test` otherwise the buttons won't appear here in the screenshot